tests/cli/test_others.py

```diff
@@ -63,12 +62,13 @@ def test_global_project_other_location(invoke, project):
 
 
 def test_uncaught_error(invoke, mocker):
-    mocker.patch.object(actions, "do_list", side_effect=RuntimeError("test error"))
-    result = invoke(["list"])
+    mocker.patch.object(actions, "do_lock", side_effect=RuntimeError("test error"))
+    result = invoke(["lock"])
     assert "[RuntimeError]: test error" in result.stderr
 
-    result = invoke(["list", "-v"])
-    assert isinstance(result.exception, RuntimeError)
+    with pytest.raises(RuntimeError) as err:
+        result = invoke(["lock"], strict=True)
+    assert "test error" in str(err)
```

Comment:
This is to test the behavior of `-v` to raise the error directly, why not keep the old way.

```python
assert isinstance(result.exception, RuntimeError)
```

Reply:
Ah ok, I think I had a problem locally.  I reverted and changed from `do_list` to `do_lock`.   Is that still ok?